src/app/components/SearchResults/SearchResults.jsx

```diff
@@ -14,7 +14,7 @@ const SearchResults = (props) => {
     <div className="nypl-row">
       <div className="nypl-column-full">
         <ResultsMetadata metadata={metadata} />
-        <ResultsList results={hits} />
+        <ResultsList results={hits} actions={props} />
```

Comment:
I would pass in `fetchWork={props.fetchWork}` instead of `actions={props}` because props includes `results` and other data you may not need or have already pass in (like `hits`). Then in `ResultsList` you can call `props.fetchWork(workId)`.